core/scripts/ocr2vrf/setup_ocr2vrf.go

```diff
@@ -225,18 +225,242 @@ func setupOCR2VRFNodes(e helpers.Environment) {
+		fmt.Println("\nFunding subscription with", subscriptionBalance, "juels...")
+		eoaFundSubscription(e, vrfCoordinatorAddress.String(), link.String(), subscriptionBalance, uint64(subID))
+	} else {
+		fmt.Println("Subscription", subID, "NOT getting funded. You must fund the subscription in order to use it!")
+	}
+
+	var forwarderAddresses []common.Address
+	var forwarderAddressesStrings []string
+	var nodesToFund []string
+	var transmitters []string
+
+	// Set up a forwarder for each node.
+	fmt.Println("Deploying transaction forwarders...")
+	for i := 0; i < *nodeCount-1; i++ {
+		// Deploy an authorized forwarder, and add it to the list of forwarders.
+		f := deployAuthorizedForwarder(e, link, e.Owner.From)
+		forwarderAddresses = append(forwarderAddresses, f)
+		forwarderAddressesStrings = append(forwarderAddressesStrings, f.String())
+	}
+	fmt.Printf("ForwarderAddresses : %v\n", forwarderAddressesStrings)
+	for i, f := range forwarderAddresses {
+
+		// Convert the sending strings for a transmitter to addresses.
+		var sendinKeysAddresses []common.Address
+		sendingKeysStrings := sendingKeys[i]
+		for _, s := range sendingKeysStrings {
+			sendinKeysAddresses = append(sendinKeysAddresses, common.HexToAddress(s))
+		}
+
+		// Set authorized senders for the corresponding forwarder.
+		setAuthorizedSenders(e, f, sendinKeysAddresses)
+
+		// Fund the sending keys.
+		nodesToFund = append(nodesToFund, sendingKeysStrings...)
+
+		// Set the authorized forwarder as the OCR transmitter.
+		transmitters = append(transmitters, f.String())
+	}
+
+	var payees []common.Address
+	var reportTransmitters []common.Address
+	for _, t := range transmitters {
+		payees = append(payees, e.Owner.From)
+		reportTransmitters = append(reportTransmitters, common.HexToAddress(t))
+	}
+
+	fmt.Printf("Setting EOA: %s as payee for transmitters: %v \n", e.Owner.From, reportTransmitters)
+	setPayees(e, vrfBeaconAddress.String(), reportTransmitters, payees)
+
+	fmt.Println("Funding transmitters...")
+	helpers.FundNodes(e, nodesToFund, big.NewInt(*fundingAmount))
+
+	fmt.Println("Deploying batch beacon consumer...")
+	loadTestConsumerAddress := deployLoadTestVRFBeaconCoordinatorConsumer(e, vrfCoordinatorAddress.String(), false, big.NewInt(*beaconPeriodBlocks))
+	addConsumer(e, vrfCoordinatorAddress.String(), loadTestConsumerAddress.String(), big.NewInt(int64(subID)))
+
+	for i := 0; i < *nodeCount; i++ {
+		// Apply forwarder args if using the forwarder.
+		if i > 0 {
+			adjustedIndex := i - 1
+			vrfJob := fmt.Sprintf(
+				cmd.Ocr2vrfTemplate,
+				vrfBeaconAddress.String(),
+				ocr2KeyBundleIDs[adjustedIndex],
+				forwarderAddresses[adjustedIndex].String(),
+				"", // P2P Bootstrapper
+				e.ChainID,
+				dkgEncrypters[adjustedIndex],
+				dkgSigners[adjustedIndex],
+				*keyID,
+				dkgAddress.String(),
+				vrfCoordinatorAddress.String(),
+				*linkEthFeed,
+			)
+			fmt.Printf("VRF JOB FOR NODE %d:\n%v\n", i, vrfJob)
```

Comment:
Lets make this a 0-based index for the non-bootstrap nodes to match our infra setup.